README.md

```diff
@@ -13,7 +13,7 @@ The canonical source code repository for libsndfile is at
 
 You can grab the source code using:
 
-    $ git clone git://github.com/erikd/libsndfile.git
+    \$ git clone git://github.com/erikd/libsndfile.git
```

Comment:
In the Github view of this markdown, the backslash remains.

Reply:
Funny, markdownlint generates [warning](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md014---dollar-signs-used-before-commands-without-showing-output). Do we need $ here?

Reply:
Yes, i just `$` without the backslash is correct,

Reply:
I mean we can remove `$` here safely. Comment from [MarkDown Style Guide](http://www.cirosantilli.com/markdown-style-guide/):

> *Don’t* prefix shell code with dollar signs $ unless you will be showing the command output on the same code block.
>
> If the goal is to clarify what the language is, do it on the preceding paragraph.
>
> Rationale: harder to copy paste, noisier to read.
>
> Good:
>
>     echo a
>     echo a > file
> Bad:
>
>     $ echo a
>     $ echo a > file
> Good, shows output:
>
>     $ echo a
>     a
>     $ echo a > file
> Good, language specified on preceding paragraph:
>
> Use the following Bash code:
>
>     echo a
>     echo a > file

Reply:
I'll check other places.